tomcat/tests/common.py

```diff
@@ -0,0 +1,61 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from datadog_checks.dev import get_here
+
+CHECK_NAME = "tomcat"
+
+HERE = get_here()
+
+"""
+These metrics cannot be added to E2E yet because they are rate and count types.
```

Comment:
Count should work just fine. Only rate, histogram and monotonic counts are expected to be missing

Reply:
just adding here that these jmxfetch metrics are monotonic_counts and this is expected to not be tested at this time.